pkgs/development/compilers/ghc/7.10.1.nix

```diff
@@ -29,6 +36,8 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   preConfigure = ''
+    cp -r ${CabalSrc}/* libraries/Cabal
```

Comment:
Yep. I figured we'd wait, but I wanted to go ahead and get your feedback about the approach.